api/src/auth/auth.module.ts

```diff
@@ -16,7 +16,7 @@ import {JwtStrategy} from "./strategies/jwt.strategy";
             imports: [ConfigModule],
             useFactory: async (configService: ConfigService) => ({
                 secret: configService.get("JWT_SECRET"),
-                signOptions: {expiresIn: "7d"}
+                signOptions: {expiresIn: "365d"}
             }),
```

Comment:
Is this this properly formatted?

Reply:
I think so, u have other idea?

Reply:
I test it, and token have expiraration time 1 year

Reply:
I meant code style

Reply:
what's wrong?

Reply:
I'm just asking if it's formatted correctly according do your code style rules. Just try to reformat code once again and if there won't be any changes - fine, nevermind.

Reply:
ok, i know what u talk about, i refactor it now.

Reply:
look, on latest commit.